test/VideoTest.js

```diff
@@ -85,4 +85,26 @@ describe('Video', () => {
         );
         expect(tag.type()).to.equal("video");
     });
+
+  it('Should support forwarding innerRef to underlying video element', function () {
+    let myRef = React.createRef();
+
```

Comment:
@maoznir shouldn't you use the forwardRef method?

Reply:
Eventually did this because it's simpler for the end user, who will be able to just do:
<Video innerRef={ref} {...props}/>

And not:

React.forwardRef((props, ref) => <Video
  innerRef={ref} {...props}
/>